pandas/tests/indexes/multi/test_indexing.py

```diff
@@ -437,3 +437,30 @@ def test_timestamp_multiindex_indexer():
     )
     should_be = pd.Series(data=np.arange(24, len(qidx) + 24), index=qidx, name="foo")
     tm.assert_series_equal(result, should_be)
+
+
+def test_get_loc_with_values_including_missing_values():
+    # issue 19132
+    idx = MultiIndex.from_product([[np.nan, 1]] * 2)
+    expected = slice(0, 2, None)
+    assert idx.get_loc(np.nan) == expected
+
+    idx = MultiIndex.from_arrays([[np.nan, 1, 2, np.nan]])
+    expected = np.array([True, False, False, True])
+    tm.assert_numpy_array_equal(idx.get_loc(np.nan), expected)
+
+    idx = MultiIndex.from_product([[np.nan, 1]] * 3)
+    expected = slice(2, 4, None)
+    assert idx.get_loc((np.nan, 1)) == expected
+
+
+def test_get_indexer_with_missing_value():
```

Comment:
testing get_loc and get_indexer are good, but we dont expect users to see those directly.  is there any user-facing behavior that this changes that should be tested?

Reply:
can you respond to this, is there any user facing change aside from .get_loc itself, IOW does user facing indexing change?

Reply:
@jreback 
`.get_slice_bound`, `__contains__`,`slice_indexer`, `slice_locs` affect from this change.
I add test case too.